lazyload_tutorials.md

```diff
@@ -394,40 +394,67 @@ spec:
 
 
 
-### Automatic ServiceFence generation based on namespace/service label
+### Support for enabling lazyload for services manually or automatically
 
-fence supports automatic generation based on label, i.e. you can define  **the scope of "fence enabled" functionality** by typing label `slime.io/serviceFenced`.
+Support for enabling lazyload for services, either manually or automatically, via the `autoFence` parameter. The configuration is as follows
 
-* namespace level
+```yaml
+---
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
+metadata:
+  name: lazyload
+  namespace: mesh-operator
+spec:
+  module:
+    - name: lazyload
+      kind: lazyload
+      enable: true
+      general:
+        autoFence: true # true for automatic mode, false for manual mode, default for manual mode
+  # ...
+```
 
-  * `true`: Servicefence cr will be created for all services (without cr) under this namespace
-  * Other values: No action
 
-* service level
 
-  * `true`: generates servicefence cr for this service
-  * `false`: do not generate servicefence cr for this service
+#### Manual mode
+
+When the `autoFence` parameter is `false`, lazyload is not automatically enabled for the services in the cluster. User is required to create the ServiceFence resource manually. This enablement is at the Service level.
+
+
+
+#### Auto mode
+
+When the `autoFence` parameter is `true`, lazyload is enabled for all services in the cluster (Note: lazyload is not enabled for services in the Namespace where Istio and Slime are located). You can also adjust the scope of lazyload by adding the Label `slime.io/serviceFenced` to the Namespace or Service. Adjustments are made at the Namespace or Service level.
```

Comment:
We need to clarify the hierarchy between `autoFence` and namespace/service level configuration, so that users will have a clearer understanding of the effect of using them together